### PR TITLE
PP-25 close to 100% density infill behaviour

### PR DESCRIPTION
**Pull request by rijkvanmanen**

### CODE REVIEW

File: resources/definitions/ultimaker.def.json [hunk 1]
```diff
@@ -43,6 +43,12 @@
         },
         "skin_angles": {
             "value": "[] if infill_pattern not in ['cross', 'cross_3d'] else [20, 110]"
+        },
+        "top_layers": {
+            "value": "math.ceil(round(top_thickness / resolveOrValue('layer_height'), 4))"
```
Comment (pkuiper-ultimaker) [inline resources/definitions/ultimaker.def.json:48]: Strange formula. The rounding does not seem necessary. Apparently this is also how it is implemented in the fdmprinters.def so I guess it is OK.

**Review by pkuiper-ultimaker — APPROVED** [review]

File: resources/definitions/ultimaker.def.json (hunk 1 above)
Comment (nallath) [reply resources/definitions/ultimaker.def.json:48]: Why wouldn't this formula be correct? It needs to be an integer.

Comment (pkuiper-ultimaker) [reply resources/definitions/ultimaker.def.json:48]: Sure, but why round first and then make it an integer.

Comment (Ghostkeeper) [reply resources/definitions/ultimaker.def.json:48]: Yeah, the `round` call seems to do nothing in this case.

**Review by nallath — APPROVED** [review]

**Pull request merged**